### ISSUE TITLE
APP source code

### ISSUE THREAD

**Issue by stoor2kushidai** [niki-amini-naieni/CountGD#26]
Thank you for your excellent work! How to use your model to count, just like an APP, is there code in the project?

**Comment by stoor2kushidai** [comment]
I want to be able to do inference counting on custom images

**Comment by niki-amini-naieni** [comment]
Yes, the CountGD computer app is available here: https://huggingface.co/spaces/nikigoli/countgd. The source code for the app is available here: https://huggingface.co/spaces/nikigoli/countgd/tree/main. The main counting function is here: https://huggingface.co/spaces/nikigoli/countgd/blob/2bb3f9251ec8101bf6cdefe83e37f4378b7c196f/app.py#L346.

**Comment by ld-xy** [comment]
> Yes, the CountGD computer app is available here: https://huggingface.co/spaces/nikigoli/countgd. The source code for the app is available here: https://huggingface.co/spaces/nikigoli/countgd/tree/main. The main counting function is here: https://huggingface.co/spaces/nikigoli/countgd/blob/2bb3f9251ec8101bf6cdefe83e37f4378b7c196f/app.py#L346.

hello ，
<img width="1062" alt="image" src="https://github.com/user-attachments/assets/c14126e2-4da7-4831-8b44-09e58362a784" />
can you help me？？thanks

**Comment by niki-amini-naieni** [comment]
Hi, can you show me the command you are running?
